docs/apache-airflow-providers-microsoft-azure/operators/azure_blob_to_gcs.rst

```diff
@@ -31,7 +31,7 @@ Please follow Azure
 to do it.
 
 TOKEN should be added to the Connection in Airflow in JSON format, Login and Password as plain text.
-You can check `how to do such connection <https://airflow.apache.org/docs/stable/howto/connection/index.html#editing-a-connection-with-the-ui>`_.
+You can check `how to do such connection <https://airflow.apache.org/docs/apache-airflow/stable/howto/connection/index.html#editing-a-connection-with-the-ui>`_.
```

Comment:
This should be internal link as it refers to another `rst` file
```suggestion
You can check :ref:`how to do such connection <howto/connection:editing-a-connection-with-the-ui>`
```

hope i got the syntax right. It's late :)
Used this [example](https://github.com/apache/airflow/blob/master/docs/apache-airflow/production-deployment.rst#impersonate-service-accounts)

Reply:
Actually we can't do this as the reference is in core doc and provider is built separatly, hence doc-build errors with reference not found